google-cloud-clients/google-cloud-firestore/src/main/java/com/google/cloud/firestore/DocumentReference.java

```diff
@@ -59,7 +59,7 @@ protected DocumentReference(
    * @return The Firestore instance.
    */
   @Nonnull
-  public Firestore getFirestore() {
+  public static Firestore getFirestore() {
```

Comment:
This would create a bug in the client, without sending any configuration(like project or instance details) user must not get a FireStore instance

Reply:
> This would create a bug in the client, without sending any configuration(like project or instance details) user must not get a FireStore instance

it will doesn't break the existing functionality